webrender/src/prim_store.rs

```diff
@@ -76,7 +71,15 @@ pub struct PrimitiveMetadata {
     pub gpu_prim_index: GpuStoreAddress,
     pub gpu_data_address: GpuStoreAddress,
     pub gpu_data_count: i32,
-    pub cache_info: Option<PrimitiveCacheInfo>,
+    // An optional render task that is a dependency of
+    // drawing this primitive. For instance, box shadows
+    // use this to draw a portion of the box shadow to
+    // a render target to reduce the number of pixels
+    // that the box-shadow shader needs to run on. For
+    // text-shadow, this creates a render task chain
+    // that implements a 2-pass separable blur on a
+    // text run.
+    pub render_task: Option<RenderTask>,
```

Comment:
I wonder if we should turn it into `Vec<RenderTask>`. Do we ever need more than one task?..